tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -837,6 +837,7 @@ def test_push_xcom_pod_info(
         self, mock_await_xcom_sidecar_container_start, mock_extract_xcom, do_xcom_push
     ):
         """pod name and namespace are *always* pushed; do_xcom_push only controls xcom sidecar"""
+        self.hook_mock.return_value.get_xcom_sidecar_container_image.return_value = "alpine"
```

Comment:
seems you should not need this change?

Reply:
@dstandish without this, the following error is raised:
`yaml.representer.RepresenterError: ('cannot represent an object', <MagicMock name='KubernetesHook().get_xcom_sidecar_container_image().to_dict()' id='139908045486608'>)`

Reply:
Oh i see, yeah, probably better to have it return None then, since here there is no connection being used, so it wouldn't return 'alpine' in this case?
```suggestion
        self.hook_mock.return_value.get_xcom_sidecar_container_image.return_value = None
```